ci/code_checks.sh

```diff
@@ -146,7 +146,7 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
     MSG='Check for inconsistent use of pandas namespace in tests' ; echo $MSG
-    for class in "Series" "DataFrame" "Index"; do
+    for class in "Series" "DataFrame" "Index" "MultiIndex" "Timestamp" "Timedelta" "TimedeltaIndex" "DatetimeIndex" "Categorical"; do
```

Comment:
didn't we move these to pre-commit?

Reply:
I don't think that's been done yet but @MarcoGorelli seemed to think it was doable

Reply:
On my to-do list :innocent:  I hope to get round to it next week